README.md

```diff
@@ -1835,12 +1835,16 @@ worksheet.addConditionalFormatting({
 
 **Cell Is Operators**
 
-| Operator    | Description |
-| ----------- | ----------- |
-| equal       | Apply format if cell value equals formula value |
-| greaterThan | Apply format if cell value is greater than formula value |
-| lessThan    | Apply format if cell value is less than formula value |
-| between     | Apply format if cell value is between two formula values (inclusive) |
+| Operator           | Description |
```

Comment:
@jbdemonte @andreykrupskii
Could you provide links to the source to confirm these changes?

Reply:
I checked, that's correct